middle.mkd

```diff
@@ -69,3 +68,12 @@ FIXME: not submitted yet
 # Security Considerations
 
 This document represents an extension to [](#RFC7230) and all security considerations therein, or in documents updating or replacing that document, also apply to this one.
+
+# Proxy considerations
+Since all Clacks messages are effectivly point-to-point transmissions there is very limited scope for multiple User-Agents to require the same message. Due to this any message with the Clacks-overhead header MUST be considered equivalent to the no-cache header defined in [RFC7234](#RFC7234) Section 5.2.1.4 this allows for the cache to store the object as long as it is revalidated prior to sending.
+
+# User-Agent considerations
+A user-agent should follow the Grand Trunk Clacks standards. For example any User-Agent recieving a Clacks-overhead message with a U code should respond to the server with the same Clacks-Overhead header on it's next request.
```

Comment:
Although I'd like to define the meaning of Clacks codes, I'd also like to complete this as soon as possible. So I think it's best to leave code definitions out of the spec rather than have an incomplete description.